src/frontend/components/LiveVideoPanel/LiveVideoTabPanel/index.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { Tab, Text } from 'grommet';
+import { defineMessages, useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+import { LivePanelItem } from 'data/stores/useLivePanelState';
+import { ShouldNotHappen } from 'utils/errors/exception';
+import { theme } from 'utils/theme/theme';
+
+interface StyledTabProps {
+  selected?: boolean;
+}
+
+const colors = {
+  active: theme.global.colors['blue-active'],
+  counterPart: theme.global.colors.white,
+  disable: theme.global.colors['blue-off'],
+  focus: theme.global.colors['blue-focus'],
+};
+
+const StyledTab = styled(Tab)`
+  background: ${colors.counterPart};
+  box-shadow: ${({ selected }: StyledTabProps) =>
+    selected
+      ? `inset 0 -2px ${colors.active}`
+      : `inset 0 -1px ${colors.disable}`};
+  flex: 1;
+  padding: 3px 10px;
+
+  div span {
+    color: ${({ selected }: StyledTabProps) =>
+      selected ? colors.active : colors.disable};
+  }
+
+  :hover {
+    box-shadow: ${({ selected }: StyledTabProps) =>
+      selected
+        ? `inset 0 -2px ${colors.active}`
+        : `inset 0 -2px ${colors.focus}`};
+
+    div span {
+      color: ${({ selected }: StyledTabProps) =>
+        selected ? colors.active : colors.focus};
+    }
+  }
+  :focus:not(:focus-visible) {
+    box-shadow: ${({ selected }: StyledTabProps) =>
+      selected
+        ? `inset 0 -2px ${colors.active}`
+        : `inset 0 -2px ${colors.focus}`};
+
+    div span {
+      color: ${({ selected }: StyledTabProps) =>
+        selected ? colors.active : colors.focus};
+    }
+  }
+`;
+
+const StyledText = styled(Text)`
+  font-family: Roboto-Bold;
+  font-size: 10px;
+  font-weight: bold;
+  letter-spacing: -0.23px;
+  text-align: center;
+  text-transform: uppercase;
+`;
+
+const messages = defineMessages({
+  application: {
+    defaultMessage: 'application',
+    description: 'Application tab title in live panel',
+    id: 'component.LiveVideoPanel.LiveVideoTabPanel.application',
+  },
+  chat: {
+    defaultMessage: 'chat',
+    description: 'Chat tab title in live panel',
+    id: 'component.LiveVideoPanel.LiveVideoTabPanel.chat',
+  },
+  joinDiscussion: {
+    defaultMessage: 'viewers',
+    description: 'Viewers and join discussion tab title in live panel',
+    id: 'component.LiveVideoPanel.LiveVideoTabPanel.joinDiscussion',
+  },
+});
+
+interface LiveVideoTabPanelProps {
+  item: LivePanelItem;
+  selected: boolean;
+}
+
+export const LiveVideoTabPanel = ({
+  item,
+  selected,
+}: LiveVideoTabPanelProps) => {
+  const intl = useIntl();
+
+  let title;
+  switch (item) {
+    case LivePanelItem.APPLICATION:
+      title = intl.formatMessage(messages.application);
+      break;
+    case LivePanelItem.CHAT:
+      title = intl.formatMessage(messages.chat);
+      break;
+    case LivePanelItem.JOIN_DISCUSSION:
+      title = intl.formatMessage(messages.joinDiscussion);
+      break;
+    default:
+      throw new ShouldNotHappen(item);
+  }
```

Comment:
An other pattern to avoid this switch and to prevent future addition in `LivePanelItem` enum is to use each enum value itself has key of the `messages` object. If you add one new value in the enum, typescript will warn you and your code will not compile anymore. 

```
const messages = defineMessages({
  [LivePanelItem.APPLICATION]: {
    defaultMessage: 'application',
    description: 'Application tab title in live panel',
    id: 'component.LiveVideoPanel.LiveVideoTabPanel.application',
  },
  [LivePanelItem.CHAT]: {
    defaultMessage: 'chat',
    description: 'Chat tab title in live panel',
    id: 'component.LiveVideoPanel.LiveVideoTabPanel.chat',
  },
  [LivePanelItem.JOIN_DISCUSSION]: {
    defaultMessage: 'viewers',
    description: 'Viewers and join discussion tab title in live panel',
    id: 'component.LiveVideoPanel.LiveVideoTabPanel.joinDiscussion',
  },
});
```

and then

```
<StyledTab
      title={<StyledText role={'tab_title'}>{intl.formatMessage(messages[item])}</StyledText>}
      plain
      selected={selected}
    />
```

WDYT ?

Reply:
I'm good with that 👍

Reply:
https://github.com/openfun/marsha/pull/1248/commits/0e6b5abc5d880e2ee9e9985c99f2b081f9a5aec2